### DISCUSSION TITLE
JavaScript interactive examples are keyboard traps

### DISCUSSION THREAD

**Discussion by stevegoldberg** [mdn/mdn-community#743] [Platform] [answered]
I was tabbing through [a page on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#try_it), and I found that once I tabbed _into_ one of the "Try It" code blocks, I could not tab out. Hitting tab indents the code in the code block, and shift+tab removes the indent.

Naturally, this is bad for accessibility as it means that keyboard users cannot continue to use the page. 

I would recommend removing the ability to indent code with tab and reserving tab for changing focus. See [WCAG](https://www.w3.org/TR/WCAG22/#no-keyboard-trap) for more details.

**Comment by pransh15** [comment]
Hi @stevegoldberg 👋 
Thanks for bringing this to light, I understand your concern. I will share it with the team and see what we can do here, and when. I'll keep you updated.

**Reply by stevegoldberg** [reply]
Cheers, thanks.

**Reply by bsmth** [reply, marked as answer]
Hi there 👋🏻 - it might be worth tracking this [in a yari issue](https://github.com/mdn/yari/issues) as it's more likely to be triaged as a bug / feature request there.

I've opened this one to track it -> https://github.com/mdn/yari/issues/12712

If you're happy with that, we can close the Discussion here so the problem / solution is easier to follow. Does that sound okay?

Thank you :)

**Reply by stevegoldberg** [reply]
If that's a better place to track the problem, then sure! I don't have a strong opinion either way, I would just like to see it get resolved :)

**Reply by bsmth** [reply]
Agreed. OK let's close for now and track progress in the issue. Thanks for raising it.
